packages/amplify_core_plugin_interface/lib/amplify_auth_category.dart

```diff
@@ -64,6 +64,11 @@ class AuthCategory {
     return plugins[0].confirmPassword(request: request);
   }
 
+  Future<AuthUser> getCurrrentUser() {
+    var request = AuthUserRequest();
+    return plugins[0].getCurrentUser();
```

Comment:
Let's suppose there's no plugin. Do you need to validate that `plugins[0]` exists? Or will that throw a range error? Or will `getCurrentException()` throw an error because it is invoked on a null reference?

Reply:
This will throw a range error - we need to discuss this as a team because we still closing on our strategy for allowing multiple plugins. I fixed this temporarily.